.github/workflows/development-ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Development to Development-CI Sync
+
+on:
+  push:
+    branches:
+      - development
+
+permissions:
+  contents: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Sync changes from development to development-ci
+
+    steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.14.1"
+
+      - name: Create development-ci branch if it doesn't exist
+        run: |
+          git branch --show-current | grep -q 'development-ci' || git branch development-ci
```

Comment:
When we first checkout the repo we are on the `development` branch so `git branch --show-current | grep -q 'development-ci' || git branch development-ci` creates a new `development-ci` branch. On the next workflow run we already have the `development-ci` branch so `git branch --show-current | grep -q 'development-ci' || git branch development-ci` will produce an error `fatal: a branch named 'development-ci' already exists` because the `--show-current` option shows the **current branch**, not all branches. Perhaps we should remove the `--show-current` option?